src/Twig/EntryFilesTwigExtension.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\WebpackEncoreBundle\Twig;
+
+use Symfony\WebpackEncoreBundle\Asset\EntrypointLookup;
+use Symfony\WebpackEncoreBundle\Asset\ManifestLookup;
+use Symfony\WebpackEncoreBundle\Asset\TagRenderer;
+use Psr\Container\ContainerInterface;
+use Symfony\Component\Asset\Packages;
+use Twig\Extension\AbstractExtension;
+
+final class EntryFilesTwigExtension extends AbstractExtension
+{
+    private $container;
+
+    public function __construct(ContainerInterface $container)
+    {
+        $this->container = $container;
+    }
+
+    public function getFunctions()
+    {
+        return [
+            new \Twig_SimpleFunction('encore_entry_js_files', [$this, 'getWebpackJsFiles']),
```

Comment:
The Twig 2.x namespace can be used here.

Reply:
yeah, it should even be used, making it future-proof (and Twig 1.x also have it)